src/main/java/herrekt/Herrekt.java

```diff
@@ -0,0 +1,89 @@
+package herrekt;
+
+import herrekt.exceptions.InvalidInputException;
+import herrekt.exceptions.NoTaskException;
+import herrekt.taskmanager.*;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+
+public class Herrekt {
```

Comment:
Generally you have few or  no comments in your code. How about including [javadocs](https://se-education.org/guides/conventions/java/basic.html#comments) as method headers to describe non-obvious functions?  Eg. `run()` seems generic, so leaving some comments about it would help  to increase understandability of your code.